doc/man7/ossl-guide-quic-client-block.pod

```diff
@@ -0,0 +1,292 @@
+=pod
+
+=begin comment
+
+NB: Changes to the source code samples in this file should also be reflected in
+demos/guide/quic-client-block.c
+
+=end comment
+
+=head1 NAME
+
+ossl-guide-quic-client-block
+- OpenSSL Guide: Writing a simple blocking QUIC client
+
+=head1 SIMPLE BLOCKING QUIC CLIENT EXAMPLE
+
+This page will present various source code samples demonstrating how to write
+a simple blocking QUIC client application which connects to a server, sends an
+HTTP/1.0 request to it, and reads back the response.
+
```

Comment:
Again I would mention that real-world servers will not support this and this is just as a demonstration of a simple protocol.